### PR TITLE
trivial spelling/grammar mistakes fixed

### PR DESCRIPTION
**Pull request by heyoni**

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](http://bugs.python.org/) (b.p.o) to verify you have signed the CLA. This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://cpython-devguide.readthedocs.io/pullrequest.html#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: pullrequest.rst [hunk 1]
```diff
@@ -271,7 +271,7 @@ request (we cannot force anyone to review pull requests and no one is
 employed to look at pull requests). If your pull request has not
 received any notice from reviewers (i.e., no comment made) after one
 month, first "ping" the issue on the `issue tracker`_ to remind the
-nosy list that the pull request needs a review.  If you don't get a response
+noisy list that the pull request needs a review.  If you don't get a response
```
Comment (Mariatta) [inline pullrequest.rst]: `nosy` is correct. Please change it back.

Comment (heyoni) [inline pullrequest.rst]: I added it back!

**Comment by Mariatta** [comment]
Thanks and congrats for your first contribution to the Dev Guide :tada:

**Pull request merged**